index.ts

```diff
@@ -206,6 +206,12 @@ export class CapabilitiesController extends BaseController<any, any> implements
    */
   getMethodKeyFor(method: string): string {
     const managedMethods: string[] = Object.keys(this.restrictedMethods);
+    const wildCardMethodsWithoutWildCard = managedMethods.reduce<{[key: string]: boolean}>(
+      (acc, methodName) => {
+        const wildCardMatch = methodName.match(/(.+)\*$/)
+        return wildCardMatch ? { ...acc, [wildCardMatch[1]]: true } : acc
+      },
+    {});
 
     // Return exact matches:
     if (managedMethods.includes(method)) {
```

Comment:
nit: this if statement should go before `wildCardMethodsWithoutWildCard` is computed

Reply:
fixed in the rebased 3b72980